### PR TITLE
Add JArray case for binding;

### PR DESCRIPTION
**Pull request by MikeStall**
support file-based scripts (like MS-Batch; Python)

### CODE REVIEW

File: src/WebJobs.Script/Binding/ExtensionBinding.cs [hunk 1]
```diff
@@ -67,13 +70,32 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes(Type para
...
+                inputValue = entityArray;
+            }
             else
             {
                 throw new NotSupportedException($"ScriptBinding type {_binding.DefaultType} is not supported");
             }
+
+            if (inputValue != null)
+            {
+                if (context.DataType == DataType.Stream)
```
Comment (mathewc) [inline src/WebJobs.Script/Binding/ExtensionBinding.cs]: The existing table binding only did this conversion when the Access was not Write (i.e. only on reads)

Comment (MikeStall) [inline src/WebJobs.Script/Binding/ExtensionBinding.cs]: I think I'd need to copy the whole if/else construct in both cases.

Comment (mathewc) [inline src/WebJobs.Script/Binding/ExtensionBinding.cs]: Just do what you need to do to unblock then, and I'll revisit soon. I was just pointing out that this method is now a little hard to understand (not your fault) and I want to investigate improvements.

**Comment by MikeStall** [comment]
merged.

**Pull request closed without merging**